### PR TITLE
Add compat data for WindowClient

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/WindowClient

as in #1002 notes on ESR might need editing.

### CODE REVIEW

**Comment by teoli2003** [comment]
I have doubt if WindowClient still exists or not. It is no more in the spec, and could have been a mixin in the first time. Marking at [DONOTMERGE} for the moment, while we dig into it a bit more.

File: api/WindowClient.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowClient",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/WindowClient.json]: This should be `"42"`.

File: api/WindowClient.json [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowClient/focus",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "42"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/WindowClient.json]: `"42"`

File: api/WindowClient.json [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowClient/focused",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "42"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/WindowClient.json]: `"42"`

File: api/WindowClient.json [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowClient/visibilityState",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "42"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/WindowClient.json]: `"42"`

**Comment by Elchi3** [comment]
> I have doubt if WindowClient still exists or not. It is no more in the spec, and could have been a mixin in the first time. Marking at [DONOTMERGE} for the moment, while we dig into it a bit more.

Like Client, WindowClient is exposed to ServiceWorkers. https://w3c.github.io/ServiceWorker/#client-interface. We still need to solve the question around how to document mixins generally (for MDN and BCD), but I don't want this to block the migration work for the moment. Moving around the data is possible and it will likely also mean moving around the MDN pages. Let's do both at the same time instead of digging into it just for BCD now.

**Pull request merged**